src/site/content/en/newsletter/archive/2019/11/index.njk

```diff
@@ -1,5 +1,6 @@
 ---
 title: Performance, Chrome Dev Summit and more
+permalink: /{{lang}}/newsletter/archive/2019/11/index.html
```

Comment:
should the /archive/ dir have an 11ty data file that sets the computed data for these so you don't have to specify the permalink for each page?

Reply:
I'm looking at it and I'm actually not sure if it's necessary anymore. I'll tinker with it to see if I can remove it, otherwise yea ill compute them instead.